public/locales/en/page.json

```diff
@@ -0,0 +1,24 @@
+{
+	"login": {
+		"login": "Login",
+		"logIn": "Login",
```

Comment:
Watch out! This was changed https://github.com/Stacha-dev/tardis-cms/blob/1d52abce17bf337cb04cc40b5e9177b45bd4305e/public/locales/cs/page.json#L6